### DISCUSSION TITLE
NSE F&O Data Symbols Suffix

### DISCUSSION THREAD

**Discussion by rajendraghelani** [pparesh25/Getbhavcopy-alternative#15] [General]
Hi Paresh,

Hope you are doing well. I am from Mumbai. 

Your work is excellent and after new implementation by NSE, We are unable to download EOD bhavcopy from anywhere. Your efforts are priceless and it works perfectly fine. 

I would like to suggest and request you to make minor changes in final F&O txt files generated by python. Earlier In Getbhavcopy and other EOD Data file download utilities, Futures Symbols were suffixed by roman figures for Near Month, Next Month & Far Month Expiries. viz. NIFTY-I, NIFTY-II & NIFTY-III. But here, symbols names are renamed as NIFTY-1, NIFTY-2 & NIFTY-3. Our old database symbol names differs from new symbol names. Is it possible to change Symbol names in final Output file? If possible, kindly update & share new updated utility.

**Comment by pparesh25** [comment]
Added py file >>> New_Nse_fo_roman_suffixes.py <<< for symbol name with roman suffixes
Also added >>> NSE_EOD_FO.zip <<< file for historical fo_eod files with symbol name with roman suffixes to fill gap in database
